.travis.yml

```diff
@@ -112,6 +112,8 @@ matrix:
         - cd flatdata-rs
         - cargo build
         - cargo test
+        - cargo build --all-features
+        - cargo test --all-features
```

Comment:
Why do we still need l113-114? Isn't it enough to have the `cargo` commands with `--all-features`?

Reply:
Yes: We want to make sure it compiles without the features enabled

Reply:
`cargo test` also does not link in contrary to `cargo build`